### PR TITLE
Create tests for technologyblip

### PR DESCRIPTION
**Pull request by A9afon**
Create more tests for technologyblip

### CODE REVIEW

File: src/test/java/com/a5lab/axion/domain/technology_blip/TechnologyBlipCfgControllerTests.java [hunk 1]
```diff
@@ -48,35 +52,15 @@ public void shouldGetTechnologyBlips() throws Exception {
...
-    ringDto.setPosition(0);
-    ringDto.setActive(true);
-
-    final TechnologyBlipDto technologyBlipDto = new TechnologyBlipDto();
-    technologyBlipDto.setId(10L);
-    technologyBlipDto.setRadar(radar);
-    technologyBlipDto.setTechnology(technologyDto);
-    technologyBlipDto.setSegment(segmentDto);
-    technologyBlipDto.setRing(ringDto);
+    final SegmentDto segmentDto = new SegmentDto(10L, null, "My segment title", "My segment description", 1, true);
```
Comment (arufanov) [inline src/test/java/com/a5lab/axion/domain/technology_blip/TechnologyBlipCfgControllerTests.java:55]: @A9afon 
* there are 2 approaches to create object

```
    final Radar radar = new Radar();
    radar.setId(10L);
    radar.setTitle("My radar");
    radar.setDescription("My radar description");

    final SegmentDto segmentDto = new SegmentDto(10L, null, "My segment title", "My segment description", 1, true);

``` 

which one we use?

File: src/test/java/com/a5lab/axion/domain/technology_blip/TechnologyBlipCfgControllerTests.java [hunk 2]
```diff
@@ -95,67 +79,120 @@ public void shouldGetTechnologyBlips() throws Exception {
...
   }
 
-  /*
+
   @Test
-  public void shouldShowTenant() throws Exception {
-    final TenantDto tenantDto = new TenantDto(10L, "my title", "my description");
-    Mockito.when(tenantService.findById(tenantDto.getId())).thenReturn(Optional.of(tenantDto));
+  public void shouldShowTechnologyBlip() throws Exception {
+    final Radar radar = new Radar();
```
Comment (arufanov) [inline src/test/java/com/a5lab/axion/domain/technology_blip/TechnologyBlipCfgControllerTests.java:85]: @A9afon 
how it is works: radar is fina object, but you able to change it by code radar.setId(10L);

**Review by arufanov — CHANGES_REQUESTED** [review]
@A9afon 
lets discuss and fix.

File: src/test/java/com/a5lab/axion/domain/technology_blip/TechnologyBlipCfgControllerTests.java (hunk 1 above)
Comment (A9afon) [reply src/test/java/com/a5lab/axion/domain/technology_blip/TechnologyBlipCfgControllerTests.java:55]: @arufanov 
| Setter | Constructor |
| ------ | --------- |
| **Advantages**: Dependencies can be changed at any time after the object has been created.  | **Advantages**: Cyclic dependency between two objects is not possible |
| **Advantages**: Оbjects of this class become suitable for reconfiguration or re-injection later. | **Advantages**: When using Spring versions 4.3+, you can completely decouple your class from a specific DI framework |
| **Advantages**: Class objects are amenable to reconfiguration and re-injection later. | **Advantages**: Allows you to implement application components as immutable objects and ensure that required dependencies are not null. Сomponents are always returned to client code in a fully initialized state. |
| **Disadvantages**: Possible circular dependency between two objects |  **Disadvantages**: The object becomes less amenable to reconfiguration and re-injection |
| **Сonclusion**: Setters should be used for optional dependency injection. | **Сonclusion**: Constructors are more suitable for mandatory dependencies and when you need immutable objects. |

I recommend using the constructor.

Comment (arufanov) [reply src/test/java/com/a5lab/axion/domain/technology_blip/TechnologyBlipCfgControllerTests.java:55]: @A9afon 
* there are a few more factors: extension of fields and long constructor created by annotation. Summary: let's discuss by voice and my assumption is settters.

File: src/test/java/com/a5lab/axion/domain/technology_blip/TechnologyBlipCfgControllerTests.java (hunk 2 above)
Comment (arufanov) [reply src/test/java/com/a5lab/axion/domain/technology_blip/TechnologyBlipCfgControllerTests.java:85]: @A9afon 
```
    final Radar radar = new Radar();
    radar.setId(10L);
    radar.setTitle("My radar");
    radar.setDescription("My radar description");
    radar = new Radar(); // !!!  Here is error due to final

```

**Review by arufanov — APPROVED** [review]

**Pull request merged**